script.js

```diff
@@ -1,9 +1,41 @@
-/* Новые элементы должны добавляться в список по нажатию на Enter */
+/*Новые элементы должны добавляться в список по нажатию на Enter */
+
+/*const sendInput = document.querySelector("#input");
+sendInput.addEventListener("keydown", function(event) {
+    if (event.key == "Enter") {
+        event.preventDefault();
+    }
+});*/
+
+const sendInput = document.querySelector("#input");
+const itemsList = document.querySelector("#items");
+
+sendInput.addEventListener("keydown", function(event) {
+    if (event.key == "Enter") {
+        // новые элементы добавляются по нажатию Enter
+
+        const messageText = sendInput.value;
+
+        const newElement = document.createElement("div");
+        newElement.classList.add("message");
+        newElement.textContent = messageText;
+
```

Comment:
You need to add event listener to newElement at the time of creation — so it will know how to behave when clicked